src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+import java.util.NoSuchElementException;
+
+public abstract class Task {
+    protected String description;
+    protected Boolean isDone = false;
+
+    public Task(String description) {
+        this.description = description;
+    }
+
+    public String getStatusIcon() {
+        return (this.isDone ? "\u2713" : "\u2718");
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public abstract String getType();
+
+    @Override
+    public String toString() {
+        return "[" + this.getStatusIcon() + "]" + this.description;
+    }
+    public Boolean isDone() {
```

Comment:
if you are trying to implement a getter and setter here for isDone variable, consider renaming your methods to setTaskStatus() and getTaskStatus(), instead of using your variable name as the method name.